### ISSUE TITLE
numb plugin not included in packages to install

### ISSUE THREAD

**Issue by vlcinsky** [ChristianChiarulli/nvcode#242]
https://github.com/ChristianChiarulli/nvcode/blob/624207db16bc44dc78c9f86078ceffd4f9f52ebd/lua/nv-numb/init.lua#L1

```lua
require('numb').setup{
   show_numbers = true, -- Enable 'number' for the window while peeking
   show_cursorline = true -- Enable 'cursorline' for the window while peeking
}
```
attempts to require `numb` but there is no such file in the repository.

**Comment by hurrtz** [comment]
you can remove/comment out `require('nv-numb')` in the `init.lua` in the project root until a fix is ready

**Comment by gauthierc** [comment]
Or add this in `lua/plugins.lua` file :
`use 'nacro90/numb.nvim'`

**Comment by ChristianChiarulli** [comment]
I think this should be fixed

**Comment by vlcinsky** [comment]
Resolved by commit c916d22e603969c5fdd7a1da19ffd6182029abc6
